### ISSUE TITLE
Remove unused SBOM dependency index fields from bombastic-index that are no longer needed.

### ISSUE THREAD

**Issue by lulf** [trustification/trustification#845]

**Comment by bxf12315** [comment]
![dependency](https://github.com/trustification/trustification/assets/209745/f00aab5d-dc77-4e57-8928-3463455a5de1)
In UI, sbom also has dependencies's count.  So can we just keep Dep's purl.?

**Comment by lulf** [comment]
Agreed, we can keep dep purl I think, it's useful to correlate. But I think other details of package is not needed. Wdyt @ctron
